File: Python/Random/random_int_between_min_max.py
import random


# Generate a random int between a given range.
def random_int_between_min_max(min, max):
    return random.randint(min, max)


def main():
    # Generate a random int between 0 and 100.
    print(random_int_between_min_max(0, 100))


main()

File: Dart/Random/random_int_between_min_max.dart
import "dart:core";
import "dart:math";

void main() {
  // Generate a random integer between 100 and 110
  print(randomIntBetweenMinAndMax(100, 110));
}

// Generate a random int between two given numbers.
int randomIntBetweenMinAndMax(final int min, final int max) {
  return Random.secure().nextInt(max - min) + min;
}
